packages/waku/src/router/client.ts

```diff
@@ -265,8 +272,7 @@ export function Link({
       };
     }
   }, [unstable_prefetchOnView, router, to]);
-  const onClick = (event: MouseEvent<HTMLAnchorElement>) => {
-    event.preventDefault();
+  const internalOnClick = (_event: MouseEvent<HTMLAnchorElement>) => {
```

Comment:
It's not confusing without it.
```suggestion
  const internalOnClick = () => {
```